modules/bbp-morphology/src/test/resources/invalid/bbp/morphology/fixstainmount/v0.1.0/wrong-type.json

```diff
@@ -1,51 +1,45 @@
 {
   "@context": [
-    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
     "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
     "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0"
   ],
   "@type": [
     "nsg:WrongType"
     ],
   "used": [{
-    "@id": "https://bbp-nexus.epfl.ch/dev/v0/data/bbp/experiment/patchedslice/v0.1.1/657e3fc4-3163-43f4-99fb-766574193a2a",
+    "@id": "{{base}}/data/bbp/experiment/patchedslice/v0.1.1/657e3fc4-3163-43f4-99fb-766574193a2a",
     "@type": [
-      "nsg:patchedSlice",
+      "nsg:PatchedSlice",
       "prov:Entity"
     ]
   }
   ],
   "generated": [{
-    "@id": "https://bbp-nexus.epfl.ch/dev/v0/data/bbp/experiment/fixedstainedslice/v0.1.1/abdcd005-e302-49c4-9874-18d1ff11c141",
+    "@id": "{{base}}/data/bbp/morphology/fixedstainedslice/v0.1.1/abdcd005-e302-49c4-9874-18d1ff11c141",
     "@type": [
-      "nsg:fixedStainedSlice",
+      "nsg:FixedStainedSlice",
       "prov:Entity"
     ]
   }],
-  "startedAtTime": {
-    "@type": "xsd:dateTime",
-    "@value": "2016-08-23T00:00:00+02:00"
-  },
+  "startedAtTime":"2016-08-23T00:00:00+02:00",
   "wasStartedBy": [
     {
       "@type": "prov:Agent",
-      "@id": "https://bbp-nexus.epfl.ch/dev/v0/data/bbp/core/person/v0.1.0/152db58a-fe71-4972-95c2-2214e32a0b15"
+      "@id": "{{base}}/data/bbp/core/person/v0.1.0/152db58a-fe71-4972-95c2-2214e32a0b15"
     }],
   "qualifiedAssociation": [{
     "@type": "prov:Association",
     "agent": {
       "@type": "prov:Agent",
-      "@id": "https://bbp-nexus.epfl.ch/dev/v0/data/bbp/core/person/v0.1.0/152db58a-fe71-4972-95c2-2214e32a0b15"
+      "@id": "{{base}}/data/bbp/core/person/v0.1.0/152db58a-fe71-4972-95c2-2214e32a0b15"
     },
     "hadRole": {
       "@type": "prov:Role",
       "@id": "HBP_ROLE:0000047"
```

Comment:
Please rename the HBP_ROLE to role and add a prefix mapping in the context for it. Let use this mapping for now
"role":"http://www.fixme.com/role_ontology"